docs/tutorial/first-app.md

```diff
@@ -109,7 +109,7 @@ const { app, BrowserWindow } = require('electron')
 
 function createWindow () {
   // Create the browser window.
-  win = new BrowserWindow({ width: 800, height: 600 })
+  let win = new BrowserWindow({ width: 800, height: 600 })
```

Comment:
Should be `const`

Reply:
Should it tho? I know it makes sense to have it as let if you only look at the snippet, but it's pretty common that you later would reassign win, for example, reassign it to null on close, which is shown later in the docs.

So I was mainly thinking that it makes more sense to have it as let, because it is let later than in the docs than having it as const and then later in the docs it all of suddenly changes to a let?

Reply:
Several times on this page for example: https://electronjs.org/docs/api/web-contents#contentssendchannel-arg1-arg2-